src/JavaScriptMinificationBundle/Resources/doc/index.rst

```diff
@@ -93,6 +78,7 @@ Usage
 - add the tag "chameleon_system.minify_js" to this service and specify an alias
 - set the configuration value `js_minifier_to_use` to this alias
 
+The bundle logs errors into channel `javascript_minify`.
```

Comment:
```suggestion
```

This doesn't really add information that isn't yet covered in the logging section. I'd remove this sentence.